lib/Slider/Slider.js

```diff
@@ -384,12 +419,33 @@ module.exports = kind(
 		this.popupContentChanged();
 		this.tapAreaClassesChanged();
 		this.initSliderStyles();
+		this.jumpIncrementChanged();
 	},
 
 	/**
 	* @private
 	*/
-	destroy: function() {
+	initComponents: function () {
+		if (this.enableJumpIncrement) this.createJumpIncrementButton();
+		ProgressBar.prototype.initComponents.apply(this, arguments);
+	},
+
+	/**
+	* @private
+	*/
+	createJumpIncrementButton: function () {
+		this.createComponents(this.jumpWrapperComponents, {owner: this});
+		this.controlParent = this.$.sliderWrapper;
+		this.removeClass('moon-slider');
```

Comment:
Hi @DavidUM, this will cause some problems when the slider is disabled from a styling perspective. We also need to disable the actual slider component (rather than the wrapper) when the control disabled, so that it cannot be spotted when disabled. The semantics were becoming confusing (see https://github.com/enyojs/moonstone/pull/2131/files#diff-90bef7cb477f9b2a2191c4de159c3017R343), as that is the slider component (or at least it has the "moon-slider" class) but its name is "sliderWrapper", while the parent control has the "moon-slider-wrapper" class. I went ahead and took a stab at a change to correct the disabled behavior, as well as renaming the component, here: https://github.com/enyojs/moonstone/compare/Enyo-1144-davidum-2.6.0-dev...ENYO-1144-aarontam. Let me know what you think, thanks!

cc @ryanjduffy